### PR TITLE
chore: pf version bump and enum fixes

### PR DESCRIPTION
**Pull request by nicolethoen**
fixes issues from https://github.com/patternfly/patternfly-react/pull/6160 and bumps patternfly/patternfly version

### CODE REVIEW

**Review by redallen — APPROVED** [review]
I was just fixing these enums now for my own PR. Thanks!

**Review by dlabrecq — APPROVED** [review]

File: packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx [hunk 1]
```diff
@@ -53,8 +52,8 @@ export interface ClipboardCopyProps extends Omit<React.HTMLProps<HTMLDivElement>
   isBlock?: boolean;
   /** Adds Clipboard Copy variant styles. */
   variant?: typeof ClipboardCopyVariant | 'inline' | 'expansion' | 'inline-compact';
-  /** Copy button popover position. */
-  position?: PopoverPosition | 'auto' | 'top' | 'bottom' | 'left' | 'right';
```
Comment (redallen) [inline packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx:57]: Hm, this is technically breaking. Maybe we can merge #6177 which appends to the types instead? Then we can rekick the core bump in #6160.

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx (hunk 1 above)
Comment (nicolethoen) [reply packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx:57]: ok

**Comment by nicolethoen** [comment]
closing in favor of #6177 and  #6160.

**Pull request closed without merging**